gen3_util/config/__init__.py

```diff
@@ -247,6 +247,21 @@ def init(config: Config, project_id: str) -> Generator[str, None, None]:
     config.state_dir = pathlib.Path(PROJECT_DIR) / 'state'
     config.state_dir.mkdir(parents=True, exist_ok=True)
 
+    state_dir_git_ignore = config.state_dir / ".gitignore"
+    if not pathlib.Path(state_dir_git_ignore).exists():
+        with open(state_dir_git_ignore, 'w') as f:
+            f.write("*\n!README.md")
+
+    meta_dir_git_ignore = "META/.gitignore"
+    if not pathlib.Path(meta_dir_git_ignore).exists():
+        with open(meta_dir_git_ignore, 'w') as f:
+            f.write("*\n!README.md")
+
+    project_readme = PROJECT_DIR + '/README.md'
```

Comment:
My notes in the issue were imprecise.

![image](https://github.com/ACED-IDP/gen3_util/assets/47808/de27d41e-5e79-4bbc-80ec-96f50a3f267f)

The intent was to create .g3t/README.md and META/README.md both with the warning not to commit files.